transport/http/outbound.go

```diff
@@ -256,6 +257,11 @@ func (o *Outbound) call(ctx context.Context, treq *transport.Request) (*transpor
 
 	span.SetTag("http.status_code", response.StatusCode)
 
+	// Service name match validation, return yarpcerrors.CodeInternal error if not match
+	if match, resSvcName := checkServiceMatch(treq.Service, &response.Header); !match {
+		return nil, transport.UpdateSpanWithErr(span, yarpcerrors.InternalErrorf("Service name not match, expected: %s, Got: %s", treq.Service, resSvcName))
```

Comment:
nit: break up this line to make it more readable. ie:
```go
return nil, transport.UpdateSpanWithErr(span,
  yarpcerrors.InternalErrorf("Service name not match, expected: %s, Got: %s", treq.Service, resSvcName))
```

Reply:
also let's fix the grammar in that message: `"service name *does* not match..."`

also note that it's preferred to lowercase error messages: https://github.com/golang/go/wiki/Errors

Reply:
Resolved. Thanks for citation!